### DISCUSSION TITLE
v4 🌈

### DISCUSSION THREAD

**Discussion by maread99** [gerrymanoim/exchange_calendars#202] [General]
# What’s Changed

## Development
* Change sessions to tz-naive, times to UTC (#179) @maread99
* v4 dev: revises \*window, bound\*, default\* and prev/next session/minute methods (#184) @maread99
* v4 dev: Rename parameters, deprecate methods and remove deprecated methods (#186) @maread99
* v4 dev: Change default side to "left" and post v4 linting (#191) @maread99

## Enhancements
* Add ExchangeCalendar.is_open_at_time (#199) @maread99

## Documentation
v4 dev: Update doc and tutorials for 4.0 (#194) @maread99

## Calendar Changes
* Removed Whit Monday and Day of German Unity holidays in 2022 (#185) @philiptromans

## Maintenance
* v4 dev: fix warnings and update trading index (#180) @maread99
* v4 dev: Advance min python version to 3.8 (#187) @maread99
* Bump actions/cache from 3.0.3 to 3.0.4 (#190) @dependabot
* Bump actions/setup-python from 3 to 4 (#193) @dependabot

## Build
* Move build meta to setup.cfg, update requirements files and workflows (#198) @maread99

Master merge #201 (released as 4.0) and subsequent #204 fix (released as 4.0.1)

**Comment by egoyret** [comment]
XLON calendar (London stock exchange) is missing the new King Charles Coronation day public holiday on May 8, 2023

**Reply by philiptromans** [reply]
It's present in `master`: https://github.com/gerrymanoim/exchange_calendars/blob/master/exchange_calendars/exchange_calendar_xlon.py#L240
